fly.toml

```diff
@@ -19,8 +19,8 @@ processes = []
   protocol = "tcp"
   script_checks = []
   [services.concurrency]
-    hard_limit = 25
-    soft_limit = 20
+    hard_limit = 250
+    soft_limit = 200
```

Comment:
@juliangruber, do you remember why you chose these particular limits? I think we should be able to handle even more than 250 req/s, maybe I can bump this up to something like 2500/2000?

Reply:
I didn't choose these limits, they were part of the default template created by the fly cli. Changing them is most likely the right step for us